src/components/FormComponents/InputField.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import { useFormStyles } from './FormComponents.styles';
+
+export const InputField = ({ type, value, callback, name, hasLabel, placeholder, width }) => {
+  const classes = useFormStyles({ width });
+  return (
+    <>
+      {hasLabel && (
+        <label htmlFor={name} className={classes.labelStyle}>
+          {name[0].toUpperCase() + name.slice(1)}
+        </label>
+      )}
+      <input
+        type={type}
+        id={name}
+        value={value}
+        onChange={callback}
+        placeholder={placeholder}
+        className={classes.inputStyle}
+      />
```

Comment:
include ...rest as prop, so that any inbuilt prop of input can be used when using InputField. 
id and name props have different purposes. use id on fragment, name for input.